More consistently describe where an annotation is allowed (#3927)
modelica/ModelicaSpecification: chapters/annotations.tex

/*literal*/ constant String defaultComponentPrefixes;
\end{lstlisting}\end{synopsis}
\begin{semantics}
Allowed for class annotations.

> Specifies a white-space separated list of recommended type prefixes to include in the \productionref{type-prefix} part of a \productionref{component-clause1} generated when creating a component of the class.

Reviewer: ```suggestion
Specifies a whitespace separated list of recommended type prefixes to include in the \productionref{type-prefix} part of a \productionref{component-clause1} generated when creating a component of the class.
```
So, no change here either.

\subsection{Version Date and Build Information}\label{version-date-and-build-information}

This section describes annotations that a top-level class can have to specify information associated to the version number.

> Some of the annotations can also be specified in the \lstinline!uses! annotation.

Reviewer: ```suggestion
Some of the annotations can also be specified in the \lstinline!uses! annotation, as stated below.
```
Or something like that.

Compared to the previous text it seemed a bit vague (which ones?)
I understand that since it is given for the specific annotations it is better to _only_ give it there - but make that clear for users.

Reviewer: I agree that it talks about what is given below. **But** it isn't obvious that the specific annotations that can be on a uses-annotations are specified below for the specific annotations, and e.g., not listed under the uses-annotation.

Just something like the following may be enough to indicate that the information is included here, and not somewhere else. Something like the following may make that clearer without restating anything:
```suggestion
Including whether they can also be specified in the \lstinline!uses! annotation.
```
